docs/source/configuration_file.md

```diff
@@ -0,0 +1,139 @@
+# Configuration file
+
+* [General parameters](#general-parameters)
```

Comment:
by referencing the file in the index.rst, the TOC will be automatically-generated, so it can be removed from here (less maintenance, less prone to errors)

Reply:
DOne :-)